res/database/Default/database_news.xml

```diff
@@ -2,6 +2,2994 @@
+			<title>
+				<de>Weltrekord auf der Schiene</de>
+				<en>World record on rail</en>
+				<pl>Rekord świata na kolei</pl>
+			</title>
+			<description>
+				<de>Im Rahmen der ICE-Weltrekordfahrt stellt der ICE-Vorläuferzug InterCityExperimental mit 406,9 km/h einen neuen Geschwindigkeitsrekord für Schienenfahrzeuge auf.</de>
+				<en>As part of the IntercityExpress world record run, the ICE predecessor model InterCityExperimental sets a new speed record for rail vehicles at 406.9 km/h.</en>
+				<pl>W ramach światowego rekordu IntercityExpress, poprzednik ICE, model InterCityExperimental, ustanowił nowy rekord prędkości dla pojazdów szynowych, osiągając 406,9 km/h.</pl>
+			</description>
+			<data genre="3" price="0.85" quality="85" fictional="0" flags="2" happen_time="4,1988,5,1" />
+		</news>
+
+			<!-- 1988-06 -->
+
+		<news id="b6872e1f-087e-4e58-ac29-78413fb5d652" type="0" creator="8936" created_by="jorgaeff" comment="https://en.wikipedia.org/wiki/Nelson_Mandela | https://www.harmonyfm.de/themen/welt-der-80er/403686-11-juni-1988-das-tribute-konzert-fuer-nelson-mandela-mit-82-acts.html | https://en.wikipedia.org/wiki/Bee_Gees | https://en.wikipedia.org/wiki/UB40 | https://en.wikipedia.org/wiki/Simple_Minds | previously 'Embley Park Stadium' but arguably doesn't need to be spoofed">
+			<title>
+				<de>Candela feiert 70. Geburtstag</de>
+				<en>Candela celebrates 70th birthday</en>
+				<pl>Candela świętuje 70. urodziny</pl>
+			</title>
+			<description>
+				<de>Im Londoner Wembley-Stadion findet anlässlich des 70. Geburtstags des westafrikanischen Politikers, Nestor Candela, ein Solidaritätskonzert statt. Namhafte Künstler wie die Ghee Bees, UB60 oder Simple Thoughts spielen für lau.</de>
+				<en>A solidarity concert is being held at London's Wembley Stadium to mark the 70th birthday of West African politician Nestor Candela. Well-known artists such as the Ghee Bees, UB60 or Simple Thoughts play for free.</en>
+				<pl>Na londyńskim stadionie Wembley odbędzie się koncert solidarnościowy z okazji 70. urodzin zachodnioafrykańskiego polityka Nestora Candeli. Znani artyści tacy jak Ghee Bees, UB60 czy Simple Thoughts zagrają za darmo.</pl>
+			</description>
+			<data genre="5" price="0.35" quality="35" fictional="0" flags="2" happen_time="4,1988,6,11" />
+		</news>
+
+			<!-- 1988-07 -->
+
+		<news id="749170dc-4248-408f-9bb5-a7d93bdcbde9" type="0" creator="8936" created_by="jorgaeff" comment="https://en.wikipedia.org/wiki/DAX | TODO: Daks vs DUX">
+			<title>
+				<de>Deutsches Aktien-System eingeführt</de>
+				<en>German stock system introduced</en>
+				<pl>Wprowadzenie niemieckiego systemu zapasów</pl>
+			</title>
+			<description>
+				<de>An der Deutschen Börse in Frankfurt wurde das Deutsche Aktien-System (Daks) eingeführt. Es misst die Wertentwicklung der 30 größten und liquidesten Unternehmen des deutschen Aktienmarktes.</de>
```

Comment:
Name mit Börsennachrichten vereinheitlichen (dort DUX)?

Reply:
yeah I noticed it too and wanted to wait for the global variables feature (and can't decide which one to use yet `:-]`)